tokio/src/runtime/context.rs

```diff
@@ -93,31 +99,54 @@ impl ThreadContext {
         self
     }
 
-    #[cfg(all(feature = "io-driver", not(loom)))]
+    #[cfg(any(
+        feature = "blocking",
+        feature = "dns",
+        feature = "fs",
+        feature = "io-std",
+        feature = "rt-threaded",
+    ))]
+    pub(crate) fn with_blocking_spawner(
```

Comment:
I think this can be replaced w/ the `cfg_blocking_impl` macro ([here](https://github.com/tokio-rs/tokio/blob/master/tokio/src/macros/cfg.rs#L24)). It cannot go around fn defs, so you would split it out to another impl block:

```rust
cfg_blocking_impl! {
    impl ThreadContext {
        pub(crate) fn with_blocking_spawner(....) { ... }
    }
}
```

Reply:
Ok, done.